psasim/tools/psa_autogen.py

```diff
@@ -49,7 +49,7 @@
         policy = "const int strict_policy[32] = { "
         qcode = "const char *psa_queues[] = { "
         versions = "const uint32_t versions[32] = { "
-        queue_path = "/tmp/psa_service_"
+        queue_path = "psa_service_"
```

Comment:
What would happen if we set the `TMP_FILE_BASE_PATH      "./somedir"` ? How will this python script pick it up?

Reply:
I kept only the filename in the python script (and in the generated C file, of course). The full path is created in `__init_psasim()` in `framework/psasim/src/service.c` by prepending `TMP_FILE_BASE_PATH`. The worst thing that can happen IMO is that the specified folder does not exists. In  this case we should adjust Makefiles to fill this gap, but I don't know if it's worth it. Wdyt?